### ISSUE TITLE
OSX py3.6: SyntaxError in wcsaxes' test_formatter_locator

### ISSUE THREAD

**Issue by bsipocz** [astropy/astropy#5717]
Errors spotted in the travis run for #5642, full details are here: https://travis-ci.org/astropy/astropy/jobs/193584174#L2424

```
==================================== ERRORS ====================================

_____ ERROR collecting lib.macosx-10.7-x86_64-3.6/astropy/tests/command.py _____

astropy/tests/command.py:13: in <module>

    from setuptools import Command

<frozen importlib._bootstrap>:961: in _find_and_load

    ???

<frozen importlib._bootstrap>:950: in _find_and_load_unlocked

    ???

<frozen importlib._bootstrap>:646: in _load_unlocked

    ???

<frozen importlib._bootstrap>:616: in _load_backward_compatible

    ???

/Users/travis/miniconda/envs/test/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/setuptools/__init__.py:170: in <module>

    ???

/Users/travis/miniconda/envs/test/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/setuptools/monkey.py:93: in patch_all

    patch_for_msvc_specialized_compiler()

/Users/travis/miniconda/envs/test/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/setuptools/monkey.py:140: in patch_for_msvc_specialized_compiler

    msvc = import_module('setuptools.msvc')

/Users/travis/miniconda/envs/test/lib/python3.6/importlib/__init__.py:126: in import_module

    return _bootstrap._gcd_import(name[level:], package, level)

<frozen importlib._bootstrap>:978: in _gcd_import

    ???

<frozen importlib._bootstrap>:961: in _find_and_load

    ???

<frozen importlib._bootstrap>:946: in _find_and_load_unlocked

    ???

<frozen importlib._bootstrap>:885: in _find_spec

    ???

<frozen importlib._bootstrap_external>:1157: in find_spec

    ???

<frozen importlib._bootstrap_external>:1131: in _get_spec

    ???

<frozen importlib._bootstrap_external>:1112: in _legacy_get_spec

    ???

<frozen importlib._bootstrap>:427: in spec_from_loader

    ???

<frozen importlib._bootstrap_external>:544: in spec_from_file_location

    ???

E     File "/Users/travis/miniconda/envs/test/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/setuptools/msvc.py", line 287

E   SyntaxError: invalid escape sequence \c

 ERROR collecting lib.macosx-10.7-x86_64-3.6/astropy/visualization/wcsaxes/tests/test_formatter_locator.py 

_pytest.runner:149: in __init__

    ???

_pytest.main:431: in _memocollect

    ???

_pytest.main:311: in _memoizedcall

    ???

_pytest.main:431: in <lambda>

    ???

astropy/tests/pytest_plugins.py:174: in collect

    module = self.fspath.pyimport()

py._path.local:650: in pyimport

    __import__(modname)

E     File "/private/var/folders/my/m6ynh3bn6tq06h7xr3js0z7r0000gn/T/astropy-test-8i2a0o_l/lib.macosx-10.7-x86_64-3.6/astropy/visualization/wcsaxes/tests/test_formatter_locator.py", line 158

E       assert fl.formatter([15.392231] * u.degree, None)[0] == "15$^\circ$23'32\""

E                                                                          ^

E   SyntaxError: invalid escape sequence \c

 ERROR collecting lib.macosx-10.7-x86_64-3.6/astropy/visualization/wcsaxes/tests/test_formatter_locator.py 

_pytest.python:606: in _importtestmodule

    ???

py._path.local:650: in pyimport

    __import__(modname)

E     File "/private/var/folders/my/m6ynh3bn6tq06h7xr3js0z7r0000gn/T/astropy-test-8i2a0o_l/lib.macosx-10.7-x86_64-3.6/astropy/visualization/wcsaxes/tests/test_formatter_locator.py", line 158

E       assert fl.formatter([15.392231] * u.degree, None)[0] == "15$^\circ$23'32\""

E                                                                          ^

E   SyntaxError: invalid escape sequence \c

== 8811 passed, 115 skipped, 72 xfailed, 1 xpassed, 3 error in 333.81 seconds ==
```

**Comment by MSeifert04** [comment]
Interesting. I thought I spotted all of them.

**Comment by bsipocz** [comment]
@MSeifert04 - maybe that was before wcsaxes got merged into core?

**Comment by astrofrog** [comment]
Fixed in f59117590364034a0cd52d056152088f8645e6cf
